Tools/Gulp/config.json

```diff
@@ -48,7 +48,6 @@
       "../../src/Cameras/Inputs/babylon.freecamera.input.keyboard.js",
       "../../src/Cameras/Inputs/babylon.freecamera.input.touch.js",
       "../../src/Cameras/Inputs/babylon.freecamera.input.deviceorientation.js",
-      "../../src/Cameras/Inputs/babylon.freecamera.input.vrdeviceorientation.js",
```

Comment:
exactly. They use the same function, so it was redundant. It is already like that in 2.4, I left it just in case there would be problems. but there weren't :smile: